### ISSUE TITLE
Team 13 - Join game

### ISSUE THREAD

**Issue by joshuahaertel** [thyer/CS340winter2016#152]
Allows you to join a finished game. Additionally colors don't align when user enters it again with a new color.

**Comment by bbarney7** [comment]
Is there a reason why joining a finished game shouldn't be allowed? What if they want to see who one?

**Comment by thyer** [comment]
Need a bit more detail on this one - does the game instantly show the "Game over!" message? Can they keep playing? Does it cause a crash?

**Comment by joshuahaertel** [comment]
It does instantly show that. You cannot keep playing and it does not crash. The original issue was that I won with one color but when I re-join with the a different color, the top bar remains the same as the first color.

Screenshot:
![screen shot 2016-03-03 at 10 10 06 am](https://cloud.githubusercontent.com/assets/16691565/13502514/857edac6-e128-11e5-9a18-49425dd67be1.png)

Additionally it looks like the data stays saved in the client. I did not notice if it was there after I won, but it was after I rejoined.

Screenshot:

![screen shot 2016-03-03 at 10 13 26 am](https://cloud.githubusercontent.com/assets/16691565/13502591/d04b0b74-e128-11e5-816f-c11cf6bbe712.png)
